electrumx/lib/tx.py

```diff
@@ -413,6 +413,48 @@ def read_header(self, height, static_header_size):
         return self._read_nbytes(header_len)
 
 
+class DeserializerGroestlcoin(DeserializerSegWit):
```

Comment:
Check the if using PR #548 will remove the need of copy-pasting code.

This should become something like this:
```
class DeserializerGroestlcoin(DeserializerSegWit):
    TX_HASH_FN = sha256
```